### DISCUSSION TITLE
status 500

### DISCUSSION THREAD

**Discussion by Hannes7c** [KelvinTegelaar/CIPP#870] [Q&A]
Good day everyone.

I have been using CIPP for about a week now and it is working for the most part. I am however receiving the below error at random times in the portal, especially when accessing tenants with over 1000 users.

Function call failure
{   "status": 500,   "data": "Function call failure",   "message": "Request failed with status code 500" }

I have checked the permissions and have granted permissions again on Azure API permissions.

Version Frontend
Current: 2.4.0
Version Backend
Current: 1.9.0 
![Error](https://user-images.githubusercontent.com/102925931/162168253-28c7c54c-31ab-43d8-ad6f-af944d587258.png)

**Comment by KelvinTegelaar** [comment]
Function call failure means it took more than 45 seconds to retrieve data, which should be really rare. I have tenants with 3000+ users that don't suffer from timeouts anywhere. 

Is your function app in a location near you? It might just be added latency due to deploying in a wrong region.

**Comment by ChrisPottrell** [comment]
Hiya Kelvin,

We're seeing this on every attempt to load BEC - does line up with the 45 second timeout you mentioned. We'll try some other tenants/users.

Chris

**Comment by Hannes7c** [comment]
Hi, will it be possible to increase the timeout? My problem is I manage tenants in multiple regions. South Africa, EU-West and UK-West. I did my deployment in EU-WEST region.
